script-library/common-debian.sh

```diff
@@ -291,7 +291,7 @@ __bash_prompt() {
         && [ ! -z "${GITHUB_USER}" ] && echo -n "\[\033[0;32m\]@${GITHUB_USER} " || echo -n "\[\033[0;32m\]\u " \
         && [ "$XIT" -ne "0" ] && echo -n "\[\033[1;31m\]➜" || echo -n "\[\033[0m\]➜"`'
     local gitbranch='`\
-        export BRANCH=$(git rev-parse --abbrev-ref HEAD 2>/dev/null); \
+        export BRANCH=$(git symbolic-ref --short HEAD 2>/dev/null || git rev-parse --short HEAD 2>/dev/null); \
         if [ "${BRANCH}" = "HEAD" ]; then \
```

Comment:
Is this block ever possible anymore, or is it safe to remove?

Reply:
Good point -  i'll take a closer look, but probably not.